### ISSUE TITLE
SMTP credential popup unclear

### ISSUE THREAD

**Issue by JosiahParry** [rstudio/blastula#248]
When creating a credential key with the below, the user received a popup saying "Enter the SMTP server password." It is unclear if this is supposed to be the password of the user email or the password of the server. It appears to be the latter—which I suspect most useRs won't be able to get their hands on. 

```
smtp_creds <- create_smtp_creds_key(
  id = "sender",
  user = "recipient"
  provider = "office365",
  host = "smtp.office365.com",
  port = 587
  use_ssl = TRUE
)
```

**Comment by andrew-fuller** [comment]
Yes I was trying to test this out on my work email in which my password was not working and I was getting the following error.
```
Error in curl_fetch_memory(smtp_server, handle = h) : Login denied
``` 
However when trying this with my personal email it worked fine.

**Comment by GISKid** [comment]
I can confirm this as well using the defaults for outlook and when specifying the port and host.
